src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -55,19 +55,12 @@ public static void clearLogStack() {
     LOGS_RESULTS.clear();
   }
 
-  private static String getPageSource(WebDriver driver) {
-    return driver.getPageSource().replaceAll("<script", "<textarea style=\"display: none\"><script")
-        .replaceAll("</script", "</script></textarea");
-  }
-
   public static void log(String command, String description, boolean success, WebDriver driver) {
     LOGS_RESULTS.add(success);
     imageCounter += 1;
 
     try {
       new Shooter().savePageScreenshot(Log.SCREEN_PATH + Log.imageCounter, driver);
-      CommonUtils.appendTextToFile(Log.SCREEN_PATH + Log.imageCounter + ".html",
-                                   Log.getPageSource(driver));
       VelocityWrapper.fillErrorLogRow(Arrays.asList(LogLevel.ERROR), description, Log.imageCounter);
     } catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

Reply:
@ludwikkazmierczak decide on one, please ;)